### ISSUE TITLE
Warning 'Float16' is only available in iOS 14.0 or newer

### ISSUE THREAD

**Issue by JanFeys** [nixberg/chacha-rng-swift#4]
I'm in Xcode Version 16.1 (16B40), iOS deployment target 16.6

When adding this package to my toy project I get the issues in the screenshot below.

<img width="1098" alt="Screenshot 2024-11-17 at 5 08 12 PM" src="https://github.com/user-attachments/assets/2208dc03-37a1-47b7-a56e-40a2483ed964">

My understanding is there are two solutions:

1. adjust the `Package.swift` to include
```    
platforms: [
        .iOS(.v14) // Minimum version that supports Float16
    ],
```
*or*

2. add `@available(iOS 14.0, *)` above each offending function

I've got limited experience so not sure how to best proceed

**Comment by nixberg** [comment]
Yeah, build was broken on Apple Silicon platforms (which I didn’t have the opportunity to test on).
Should be fixed in `0.8.1`.

**Comment by JanFeys** [comment]
Thanks, it is indeed resolved in 0.8.1
